okhttp/src/main/kotlin/okhttp3/internal/concurrent/LoomBackend.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.concurrent
+
+import java.lang.invoke.MethodHandle
+import java.lang.invoke.MethodHandles
+import java.lang.invoke.MethodType
+import okhttp3.internal.SuppressSignatureCheck
+import okhttp3.internal.notify
+
+@SuppressSignatureCheck
+class LoomBackend() : Backend {
```

Comment:
We may want to implement this closer to the RealBackend example using

Executors.newUnboundedVirtualThreadExecutor()

Or Thread.Builder?